### PR TITLE
feature: adding emacsti-commit-after-save

### PR DESCRIPTION
**Pull request by Malnati**
I just added a function called git push --set-upstream origin feat/emacsti-commit-after-save, and I did a hook for calling after each saves, as well. 
After this point, it will commit to git repository every time Emacs saves a file. I have used a bash execution called by Elisp function, and I have tested it in Linux only.

### CODE REVIEW

File: save.el [hunk 1]
```diff
@@ -62,3 +62,11 @@
...
 (setq yank-menu-length 9999)
 (message "→ yank-menu-length 9999!")
+
+(defun emacsti-commit-after-save ()
+       "Commit all buffers to git after save C-x C-s."
+       (interactive)
+       (shell-command-to-string
+	      "git add . && git commit -m \"Auto-commited by emacsti-commit-after-save\"")
+       (message "Exceuted → Auto-commited by emacsti-commit-after-save"))
+(add-hook 'after-save-hook #'emacsti-commit-after-save)
```
Comment (mendesroberto) [inline save.el:72]: Oi Ricardo,

Tive uma idéia de criar um alias para invocar o git-push. Dessa forma não estaríamos acionando a todo instante o repositório.

Comment (Malnati) [reply save.el:72]: I have tried checking if It stays in a git project using the command below.  However, I guess we could use more time before applying those features as a hook.
`git -C ~/ rev-parse; ITISGITREPO=$?; test $ITISGITREPO -eq 0 && echo " IF ITISGITREPO => $ITISGITREPO"`

Comment (Malnati) [reply save.el:72]: Let's add a hook or keystroke at custom.el and we can decide after some use of this solution.

**Review by mendesroberto — CHANGES_REQUESTED** [review]
Please, make the requested changes.

**Review by mendesroberto — CHANGES_REQUESTED** [review]
Please, add the hook in the README.md.

**Review by mendesroberto — APPROVED** [review]
Approved.

**Pull request merged**